scripts/convert_lit_checkpoint.py

```diff
@@ -123,7 +123,7 @@ def convert_lit_checkpoint(
 
     # checkpoint_name cannot be hardcoded because there exists different outputs such as
     # ("lit_model_finetuned.pth", "lit_model_lora_finetuned.pth", "lit_model_adapter_finetuned.pth"")
-    pth_file = checkpoint_dir / checkpoint_name
+    pth_file = checkpoint_dir / "".join([checkpoint_name, ".pth"])
```

Comment:
Why hardcode the extension here instead of letting the user pass it in the checkpoint name?

`--checkpoint_name=foo.whatever`

Reply:
the finetuning scripts save to .pth files, so I am really just saving users a need to include .pth. I'm indifferent on this and will change it if you'd like.

Reply:
updated in [e716629](https://github.com/Lightning-AI/lit-gpt/pull/321/commits/e7166299f39e714dd64e2ee9d4e8f7faec8b2775)

Reply:
I wouldn't do this, in case the extension is renamed or something similar. Seeing the extension can be useful to know exactly what you are meant to pass.